calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -45,13 +45,15 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 
 - Windows
 - eBPF dataplane
+- GKE clusters
 - RKE and RKE2 clusters
 
 **Limitations**
 
 - WAF is not supported for host-networked client pods
 - When selecting and deselecting traffic for WAF, active connections may be disrupted
-- You cannot use the annotation on [LoadBalancer type](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) services. 
+- You cannot use the annotation on [LoadBalancer type](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) services.
```

Comment:
Not sure what "use the annotation" means. User doesn't have a clue at this point in the doc what this refers to. 
- You cannot use the annotation on [LoadBalancer type](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) services.

Reply:
I'm honestly not sure myself - this was already in the file I edited but it isn't on the docs page for WAF